src/main/kotlin/com/google/protobuf/gradle/ProtobufConfiguratorExts.kt

```diff
@@ -24,12 +24,12 @@ import org.gradle.kotlin.dsl.invoke
  * ```
  *
  * @receiver [Project] The project for which the plugin configuration will be applied
- * @param action A configuration lambda to apply on a receiver of type [ProtobufConfigurator]
+ * @param action A configuration lambda to apply on a receiver of type [ProtobufExtension]
  *
  * @return [Unit]
  */
-fun Project.protobuf(action: ProtobufConfigurator.()->Unit) {
-    project.convention.getPlugin(ProtobufConvention::class.java).protobuf.apply(action)
+fun Project.protobuf(action: ProtobufExtension.() -> Unit) {
```

Comment:
When we remove the extension registration from afterEvaluate, we can remove that extension function.